src/components/Banner/FeatureBanner.tsx

```diff
@@ -11,11 +12,18 @@ const Buttons = (props: { scrollTo: any }) => {
         color="primary"
         disableRipple
         disableFocusRipple
-        onClick={() => props.scrollTo()}
+        onClick={() => {
+          trackEvent(
+            EventCategory.INDIGENOUS_PEOPLES_DAY,
+            EventAction.CLICK,
+            'View chart',
```

Comment:
Would it make any sense to do "Banner: View chart" or something to make it clearer that this click is related to the homepage banner?  Up to you.